src/models/MainWeatherDetails.tsx

```diff
@@ -0,0 +1,23 @@
+export default class MainWeatherDetails {
+
+    temp: number;
```

Comment:
This name `temp` is making me to think about something temporary 😂 Can we use another name for it? i.e. `temperature`